app/src/main/res/values/strings.xml

```diff
@@ -157,8 +157,8 @@
     <string name="moveDown">Move downwards</string>
     <string name="leaveWithoutSaveTitle">Exit</string>
     <string name="leaveWithoutSaveConfirmation">Leave without saving?</string>
-    <string name="addManually">Manually enter ID</string>
-    <string name="addFromImage">Select image from gallery</string>
+    <string name="addManually">Select the barcode manually</string>
```

Comment:
To me, "select the barcode manually" sounds like it's missing the "enter the text/ID" part.

```suggestion
    <string name="addManually">Input the barcode manually</string>
```